scripts/generate/templates/PRODUCT_SUBSCRIPTION/react.template.js

```diff
@@ -217,8 +219,8 @@ function Remove() {
     <>
       <Text size="titleLarge">{localizedStrings.hello}!</Text>
       <Text>
-        Remove {`{Product id ${data.productId}}`} from{' '}
-        {`{Plan group id ${data.sellingPlanGroupId}}`}
+        Remove Product id {data.productId} from Plan group id{' '}
```

Comment:
Why do we need `{' '}`

Reply:
The formatter adds it to ensure there is an obvious space there.